test/AccountCreation.ui.ts

```diff
@@ -475,13 +443,16 @@ describe('Account creation', function () {
             await accountNameInput.sendKeys(ACCOUNTS.ANY);
             await continueBtn.click();
 
-            const importSuccessForm = await this.driver.wait(
-              until.elementLocated(By.css('[data-testid="importSuccessForm"]')),
-              this.wait
-            );
-            expect(importSuccessForm).not.to.be.throw;
+            expect(
```

Comment:
Not sure what this `expect` actually checks. @smelnikov ? As I understand this should check if provided function throws on invocation, but we don't have a function here

https://www.chaijs.com/api/bdd/#method_throw

And there are many other checks like this

Reply:
When some transition comes, some pages/components are not reachable by `this.driver.findElement`, so we need to wait for them with `this.driver.wait`.
The original idea was that in some cases we need to check that an element appears successfully without breaking the test.
Now this check seems unnecessary.